### PR TITLE
Update splice docs

### PR DESCRIPTION
**Pull request by tasawar-hussain**
Make the description more readable and understandable

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/array/splice/index.html [hunk 1]
```diff
@@ -128,7 +128,7 @@ <h3 id="Remove_2_elements_from_index_0_and_insert_parrot_anemone_and_blue">Remov
 // myFish is ["parrot", "anemone", "blue", "trumpet", "sturgeon"]
 // removed is ["angel", "clown"]</pre>
 
-<h3 id="Remove_2_elements_from_index_2">Remove 2 elements from index 2</h3>
+<h3 id="Remove_2_elements_before_index_2">Remove 2 elements bfore index 2</h3>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/array/splice/index.html:131]:
```suggestion
<h3 id="Remove_2_elements_before_index_2">Remove 2 elements before index 2</h3>
```

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/javascript/reference/global_objects/array/splice/index.html (hunk 1 above)
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/array/splice/index.html:131]:
```suggestion
<h3 id="Remove_2_elements_before_index_2">Remove 2 elements before index 2</h3>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Tasawar, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Comment by wbamberg** [comment]
I'm not sure about this. This example: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice#remove_2_elements_from_index_2 seems to be removing 2 elements *starting at* index 2:

```js
let myFish = ['parrot', 'anemone', 'blue', 'trumpet', 'sturgeon']
let removed = myFish.splice(2, 2)
myFish        // [ "parrot", "anemone", "sturgeon" ]
```

I'm not sure "Remove 2 elements *before* index 2" is a correct way to describe this. "Remove 2 elements *starting from* index 2" might be an improvement though.

Or am I confused here?

**Comment by sideshowbarker** [comment]
> I'm not sure "Remove 2 elements _before_ index 2" is a correct way to describe this. "Remove 2 elements _starting from_ index 2" might be an improvement though.
> 
> Or am I confused here?

You’re not confused. I should’ve been more careful in reviewing this. I’ve opened https://github.com/mdn/content/pull/2679 with a fix.

**Pull request merged**